### ISSUE TITLE
[MultiComboBox|A11y]: Irrelevant info is read out on opening drop-down

### ISSUE THREAD

**Issue by f16r** [SAP/ui5-webcomponents#9116]
### Bug Description

After opening the drop-down with the key combination Alt + Arrow Down JAWS reads the title of the document with suffix "Google Chrome Dialog" (e.g. MultiComboBox | UI5 Web Components - Google Chrome dialog on https://sap.github.io/ui5-webcomponents/components/MultiComboBox/).

![image](https://github.com/SAP/ui5-webcomponents/assets/37051913/b811f4e4-b8a2-4679-8452-df2ba3f91b75)

### Affected Component

MultiComboBox

### Expected Behaviour

Use a more meaningful name that links the dialog to the multi combobox input instead to the overall document.

### Isolated Example

https://sap.github.io/ui5-webcomponents/components/MultiComboBox/

### Steps to Reproduce

1. Press Alt + Arrow Down

### Priority

None

### UI5 Web Components Version

1.24.1

### Browser

Chrome

### Organization

CBC

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by elenastoyanovaa** [comment]
Hello @f16r ,

This seems to be a JAWS specific behavior - when a dialog is not labelled (in our case the multicombobox popover) the Screen Reader seems to announce the `<title>` element of the html document as a dialog title, on opening the dialog. 
In order to avoid this behavior I have applied a label to the suggestions popup. Additionally I did the same for the ui5-input and ui5-combobox as well. 

Thank you for identifying this behavior and helping us improve the accessibility of our components!

The issue will be closed automatically once the improvement is merged in the release 1.24 branch.

Kind Regards,
Elena

**Comment by elenastoyanovaa** [comment]
Hello,

The fix will be available with the next release of 1.24.

Kind Regards,
Elena
